### PR TITLE
Improve default topic categorisation of passes

### LINKED ISSUE #75: Improve initial categorisation
currently even if they are expired they end up in new

context: https://play.google.com/apps/publish/?account=5229070112536767347#ReviewDetailsPlace:p=org.ligi.passandroid&reviewid=gp:AOqpTOHvau--GuU29tG4Qo93sSXC0q_4cZixEtMLyd-ndBIqrzpCczxFcYlnw-w9tchWZnq12pY_dHXBPJ4ZR9Y

### PR DESCRIPTION
**Pull request by andrewlord1990**
Sorry, I screwed up the commit history a bit when trying to rebase everything as a single commit. So I have recreated the changes and opened a new PR.

Fixes #75

Previously the initial topic for a pass was simply "new".

After this change, if the pass was either in the pass (calendar timespan) or has expired (valid timespan) then it will have an initial topic of "archive".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ gen
 project.properties
 gradle.properties
 local.properties
+*iml
+/build.gradle
```
Comment (ligi) [inline .gitignore]: why add ignore for this file?

Comment (andrewlord1990) [inline .gitignore]: Whenever I synced to the gradle file a root build.gradle was created and the various .iml files. I thought this would affect other people too, so added them to the .gitignore. I probably shouldn't include this in a PR as an unrelated change though. Sorry, happy to take it out of this branch if you would like me to.

**Comment by ligi** [comment]
ack - will merge this PR but have to experiment with this a bit more.
Not yet sure about the UX - when e.g. we found only old passes - there
will be a notification that there are "new" ones but none are in the new
folder. I think the notifications have to be adapted in the same time -
can look at this deeper tomorrow

**Comment by andrewlord1990** [comment]
Yeah I agree. The notification code would need to only notify when there are any found in the topic "New". However, currently this happens anyway, its just they aren't really New. So for example I reinstalled the app recently and now I have all the cinema tickets I have ever had in "New". So I think this PR improves one aspect of it, which is moving them to Archive automatically. The next step is to improve the notification. I am happy to investigate improving this area too.

**Comment by ligi** [comment]
:+1: thanks
